quipucords/api/signal/scanjob_signal.py

```diff
@@ -98,12 +103,17 @@ def scan_restart(sender, instance, **kwargs):
     instance.log_message(_(messages.SIGNAL_STATE_CHANGE) % ('RESTART'))
     scanner = ScanJobRunner(instance)
 
-    if not SCAN_MANAGER.is_alive():
-        SCAN_MANAGER.start()
+    if not manager.SCAN_MANAGER.is_alive():
+        logger.error('%s: Process unexpectedly crashed.  See logs.',
```

Comment:
Not a blocker, but we should consider putting these into a messages file since we reuse the same message.